### PR TITLE
update setup.sh for human-dynamics profile

### PR DESCRIPTION
**Pull request by lrapetti**
The human-dynamics profile install the [`human-gazebo`](https://github.com/robotology/human-gazebo) models, but the path to those models need to be add to `YARP_DATA_DIRS` in order to be found. For this reason I add this path in the `setup.sh`

### CODE REVIEW

**Comment by lrapetti** [comment]
@kouroshD is there any other path that needs to be add for the `human-dynamics` profile?

**Comment by traversaro** [comment]
This is exactly the reason why I would prefer if `YARP_DATA_DIRS` could be avoid to use, see https://github.com/robotology/yarp/issues/336 , otherwise for each YARP "package" (i.e. something that installs in `<prefix>/share`) a new directory needs to be appended to `YARP_DATA_DIRS`. 

However, this is not directly related to this PR. To finish the job, can you also: 
* Update the readme docs in https://github.com/robotology/robotology-superbuild#configuration-3 (i.e. the configuration of the human dynamics component)
* Update the scripts to add/remove the paths from the persistent Windows enviroment variables, see https://github.com/robotology/robotology-superbuild/blob/master/cmake/template/removePathsFromUserEnvVariables.ps1.in and https://github.com/robotology/robotology-superbuild/blob/master/cmake/template/addPathsToUserEnvVariables.ps1.in

**Comment by lrapetti** [comment]
> To finish the job, can you also:
> 
> * Update the readme docs in https://github.com/robotology/robotology-superbuild#configuration-3 (i.e. the configuration of the human dynamics component)

I think this was already handled with https://github.com/robotology/robotology-superbuild/pull/257/commits/9c6f13931d9e380facc214da75b45419fbe28273 

> * Update the scripts to add/remove the paths from the persistent Windows enviroment variables, see https://github.com/robotology/robotology-superbuild/blob/master/cmake/template/removePathsFromUserEnvVariables.ps1.in 

https://github.com/robotology/robotology-superbuild/pull/257/commits/83d217f4e9dda215aafe4960dfd05587b81727c5

> * and https://github.com/robotology/robotology-superbuild/blob/master/cmake/template/addPathsToUserEnvVariables.ps1.in

https://github.com/robotology/robotology-superbuild/pull/257/commits/724228bbff2993dfd58bdb54884430c7f2facc2e

File: README.md [hunk 1]
```diff
@@ -487,8 +487,7 @@ Install matio following the [installation instructions](https://github.com/tbeu/
 
 ### Configuration
 `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/codyco` must be appended to the `YARP_DATA_DIRS` enviromental variable.
-If you are using Linux or macOS, the `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/robotology-superbuild/setup.sh` script will append the necessary path to `YARP_DATA_DIRS`.
-
+The scripts `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/robotology-superbuild/setup.sh` (Linux, MacOs) or `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/robotology-superbuild/setup.bat` (Windows), will append the necessary path to `YARP_DATA_DIRS`.
```
Comment (traversaro) [inline README.md]:
```suggestion
The scripts `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/robotology-superbuild/setup.sh` (Linux, macOS) or `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/robotology-superbuild/setup.bat` (Windows), will append the necessary path to `YARP_DATA_DIRS`.
```

File: README.md [hunk 2]
```diff
@@ -543,9 +542,13 @@ This profile is enabled by the `ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS` CMake option.
 ### System Dependencies
 To run a human dynamics estimation scenario, we need a Windows machine to install the Xsens suit SDK for getting the sensory information of the human motions from [Xsens](https://www.xsens.com/) and [ESD USB CAN driver](https://esd.eu/en/products/can-usb2) to get the FTShoes/FTSkShoes sensory information. Refer to [Xsens](#xsens) and [FTShoes/FTSkShoes](#shoes) for more information about the dependencies.
 
-### Configuration
+### System Dependencies
 The steps necessary to install the system dependencies of the Human Dynamics profile are provided in operating system-specific installation documentation, and no additional system dependency is required.
 
+### Configuration
+`$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/human-gazebo` must be appended to the `YARP_DATA_DIRS` enviromental variable.
+The scripts `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/robotology-superbuild/setup.sh` (Linux, MacOs) or `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/robotology-superbuild/setup.bat` (Windows), will append the necessary path to `YARP_DATA_DIRS`.
```
Comment (traversaro) [inline README.md]:
```suggestion
The scripts `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/robotology-superbuild/setup.sh` (Linux, macOS) or `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/share/robotology-superbuild/setup.bat` (Windows), will append the necessary path to `YARP_DATA_DIRS`.
```

File: README.md [hunk 3]
```diff
@@ -543,9 +542,13 @@ This profile is enabled by the `ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS` CMake option.
 ### System Dependencies
 To run a human dynamics estimation scenario, we need a Windows machine to install the Xsens suit SDK for getting the sensory information of the human motions from [Xsens](https://www.xsens.com/) and [ESD USB CAN driver](https://esd.eu/en/products/can-usb2) to get the FTShoes/FTSkShoes sensory information. Refer to [Xsens](#xsens) and [FTShoes/FTSkShoes](#shoes) for more information about the dependencies.
 
-### Configuration
+### System Dependencies
```
Comment (traversaro) [inline README.md]: For some reason the `System Dependencies` has been duplicated.

Comment (lrapetti) [reply README.md]: My bad, the information in this replicated System Dependencies previously were in the **Configuration** section (even if they refering to system dependency). I think we can remove that part.

Comment (lrapetti) [reply README.md]: removed https://github.com/robotology/robotology-superbuild/pull/257/commits/da2baa151f9f0d933c7023c862228c1dda4ff34c

**Comment by traversaro** [comment]
Thanks!

**Pull request merged**